index_add

mamiko-7172 opened the pull request:
indexのHTMLを記入しました

mamiko-7172 committed:
index_add

indexのHTMLを記入しました

ArisaNAKAMURA-GitHub commented on resources/views/index.blade.php:
```diff
@@ -0,0 +1,62 @@
+            <div class="title">
+                <h1>検索結果</h1>
+            </div>
+            <div class="sarch">
```
［nits(typo)］クラス名 、"search"だと思われます。

ArisaNAKAMURA-GitHub commented on resources/views/index.blade.php:
```diff
@@ -0,0 +1,62 @@
+                <h1>検索結果</h1>
+            </div>
+            <div class="sarch">
+                <p>検索条件をリセット</p>
```
［IMO］リセットだったらどこかにページ飛ばすようになるのでaタグをつけた方がいいかなと思います。

ArisaNAKAMURA-GitHub commented on resources/views/index.blade.php:
```diff
@@ -0,0 +1,62 @@
+                <a href=""><img src="" alt="お気に入り星"></a>
+                <a href=""><img src="" alt="口コミ鉛筆"></a>
+                <div class="ward">
+                    <p>口コミ投稿する</p>
```
［nits］デザインに忠実に書くなら「口コミ”を”投稿する」になると思います。

ArisaNAKAMURA-GitHub commented on resources/views/index.blade.php:
```diff
@@ -0,0 +1,62 @@
+                <p>検索条件をリセット</p>
+            </div>
+            <div class="choice">
+                <p>指定条件</p>
```
［ask］確認です。この後に検索した時の指定条件を呼び出すコードを書く感じですかね？

mamiko-7172 replied:
そうです。一旦この記載にしていました。分かりやすく書き直しておきます。

ArisaNAKAMURA-GitHub commented on resources/views/index.blade.php:
```diff
@@ -0,0 +1,62 @@
+        </div>
+        <div class="review">
+            <div class="storephoto">
+                <img src="" alt="店舗写真">
```
［IMO］記憶違いかもしれませんが、店舗写真からも店舗ページに飛ぶことができたように思います。加えてaタグが必要かもしれません。

mamiko-7172 replied:
そうでしたね！aタグで飛ばすようにしておきます。

ArisaNAKAMURA-GitHub commented on resources/views/index.blade.php:
```diff
@@ -0,0 +1,62 @@
+            </div>
+            <div class="add">
+                <div class="ward">
+                    <p>お気に入り追加</p>
```
［nits］デザインに忠実に書くなら「お気に入り”に”追加」になると思います。

ArisaNAKAMURA-GitHub commented on resources/views/index.blade.php:
```diff
@@ -0,0 +1,62 @@
+                <img src="" alt="店舗写真">
+            </div>
+            <div class="add">
+                <div class="ward">
```
［nits(typo?)］ 36行目もですが、もしかして”word”ですか？

ArisaNAKAMURA-GitHub commented:
細かいとことか確認事項が多めですみません…。
CSS作成時とかphpコードを書く時にまたちょっとずつdivタグとかpタグが変更になる感じですかね？
大まかなカラムの分け方というか構造に関して、見やすくて良いと思いました！作成ありがとうございます！！

mamiko-7172 committed:
indexのコード確認してもらった後に修正

中村さんにコード確認してもらった点を修正しました

mamiko-7172 committed:
iindexのプラスで修正しました

mamiko-7172 committed:
indexに関しての追加

・SweetsControllerの追加
・web.phpにindexルートを作成
・publicにcssの追加
　→style.css、addreview.jpg、okiniiri.pgを追加
・index.phpに追記

chihiro0415 commented:
controller、井上さんがあげてくれていたものを使って
StoreController@index を使うと良さそうですー！

mamiko-7172 commented:
@chihiro0415 
controllerの件、承知しました！！マージするときには消してStoreController@index使うようにします！

mamiko-7172 committed:
indexページに関して

・SweetControllerの削除
・web.phpからもSweetControllerのコネクト記載を悪女
・style.cssへの追記
・index.phpへの追記、変更
　@foreachの記入（有効にはしていない）
　DBから引っ張ってくるコードを変更

mamiko-7172 committed:
Merge branch 'master' into add_index

mamiko-7172 committed:
indexに関しての追記
・StoreControllersのindexクラスに追記
・web.phpにrouteを追記
・index.phpの記載の変更（データ引っ張ってない状態にしました）
・style.cssに追記

tai-namechan reviewed (approved):
確認できました！！LGTM!!
